src/collapse/dtrace.rs

```diff
@@ -182,8 +187,17 @@ impl Folder {
             frame = Self::uncpp(frame);
         }
 
-        if frame.is_empty() {
-            frame = "-";
+        let frame = if frame.is_empty() {
+            Cow::Owned("-".to_owned())
+        } else if self.opt.demangle {
+            let mut parts = frame.split('`');
```

Comment:
Are there only ever two parts to this? Or do we want something like `rsplitn(..., 2)`?

Reply:
I believe there is only ever one `` ` ``, but this made me think of something else. If the `includeoffset` option is used this probably won't work in its current state since it will have an offset, such as `+0x21`, at the end. I'll change it to handle this case.